src/streams/readStream.ts

```diff
@@ -33,12 +33,12 @@ declare module "../Client" {
     /**
      * Sends events to a given stream.
      * @param streamName A stream name.
-     * @param count Amount to read
+     * @param maxCount Amount to read
      * @param options Reading options
      */
     readStream(
       streamName: string,
-      count: number | BigInt,
+      maxCount?: number | BigInt,
```

Comment:
The general pattern for the client is: 

`command(...requiredArgs, { ...optionalArgs })` 

So this should be moved into `ReadStreamOptions` if it is to become optional.

Reply:
Moved in https://github.com/EventStore/EventStore-Client-NodeJS/pull/130/commits/5e42ba746c53554e09e27bdd5e3c81bf93332d05.